backend/src/graphql/model/UserAdmin.ts

```diff
@@ -14,7 +16,7 @@ export class UserAdmin {
     this.hasElopage = hasElopage
     this.deletedAt = user.deletedAt
     this.emailConfirmationSend = emailConfirmationSend
-    this.isAdmin = user.isAdmin
+    this.roles = user.userRoles?.map((userRole) => userRole.role) ?? []
```

Comment:
``user.userRoles?`` immer Wahr!?

Reply:
Does not work without this notation

Reply:
yes Moritz is right, theoretically is is always true, but it does not work without this notation